kafka/modis_kernels_kf.py

```diff
@@ -229,21 +230,22 @@ def get_mcd43_prior(mcd43_fstring, band):
                 gdal.Open(os.path.join(the_dir, "brdf_2016_b06.vrt")),
                 gdal.Open(os.path.join(the_dir, "brdf_2016_b07.vrt")))
 
-    output_plethora = create_output_file(g, the_dir, "test")
+    output_plethora = create_output_file(g, the_dir, "band{}".format(band))
     
-    kf = MODISKernelLinearKalman(modis_obs, days, output_plethora, [] )
+    kf = MODISKernelLinearKalman(modis_obs, days, output_plethora, [], the_band=band )
```

Comment:
I've done a quick grep for self.the_band and it is only used in modis_kernels_kf and only to read the data as above and for the uncertainties - currently, to read from data band counts from 1 not zero but will be wrong for the uncertainty. so currently they are inconsistent. I'd prefer to put a band-1 for the uncertainty to make it consistant - otherwise there will be a lot od band+1 everywhere else and it will be less intuitive (since the modis bands are typically called 1 through 6). Will change and submit.